docs/content/guides/dagster/parallel-computing-dask.mdx

```diff
@@ -0,0 +1,270 @@
+
+- Set up a Dask resources
+- Use Dask to speed up data assets
+- Use Dask’s machine learning tools to build a model
+
+---
+
+## Using Dagster’s resources to set up a Dask Cluster
+
+When using Dask with Dagster, you can use the `DaskResource` to set up a Dask client and simplify a pipeline that uses Dask across multiple assets. In this example, a local`dask.distributed` client is set up as a `ConfigurableResource`. Resources allow you to centralize your configuration in a single location, reducing redundant code in your assets.
+
+```python file=/guides/dagster/parallel-computing-dask/resources.py startafter=first_resource_start endbefore=first_resource_end
+from dask.distributed import Client
+
+# isort: skip_file
+from dagster import ConfigurableResource
+
+
+class DaskResource(ConfigurableResource):
+    def make_dask_cluster(self) -> Client:
+        client = Client()
+        return client
+```
+
+The `dask.distributed.LocalCluster` API allows customization of the cluster, which can be leveraged through Dagster’s `resource`. In the following example, configuration is setup at the resource level, which is then leveraged by Definitions and applied to all uses of the resource.
+
+```python file=/guides/dagster/parallel-computing-dask/resources.py startafter=second_resource_start endbefore=second_resource_end
+from dagster import ConfigurableResource, Definitions
+from dask.distributed import Client, LocalCluster
+
+
+class DaskResource(ConfigurableResource):
+    n_workers: int
+
+    def make_dask_cluster(self) -> Client:
+        client = Client(LocalCluster(n_workers=self.n_workers))
+        return client
+
+
+defs = Definitions(
+    assets=[asset_1, asset_2], resources={"MyDaskResource": DaskResource(n_workers=2)}
+)
+```
+
+In the following example, the `LocalCluster` configuration is being used in `make_dask_cluster`, which can be set for each `@asset`.
+
+```python file=/guides/dagster/parallel-computing-dask/resources.py startafter=third_resource_start endbefore=third_resource_end
+class DaskResource(ConfigurableResource):
+    def make_dask_cluster(self, n_workers) -> Client:
+        client = Client(LocalCluster(n_workers))
+        return client
+
+
+@asset
+def resource_asset(MyDaskResource: DaskResource):
+    return MyDaskResource.make_dask_cluster(n_workers=5)
+```
+
+---
+
+## Using the Dask Resource
+
+Now that the Dask resource is set up, an asset can use it to parallelize computations. This asset demonstrates using the `client` within Dagster using the example from the [Dask distributed quickstart](https://distributed.dask.org/en/stable/quickstart.html).
+
+```python file=/guides/dagster/parallel-computing-dask/use_dask.py startafter=first_ex_start endbefore=first_ex_end
+from dagster import asset, MetadataValue, Definitions
+
+
+@asset
+def simple_dask_asset(context, MyDaskResource: DaskResource):
```

Comment:
variable names should be lower case, maybe just `dask_resource` or `dask`

Reply:
it looks like `context` isn't used, so best to just omit it

Reply:
including a return type annotation would help make it clear what this function is doing

also, does our default IO manager work with dask dataframes?